TicTacToe Project

dfobastos opened the pull request:
Primeiros dois passos completos:

- Introduzir o nome do jogador 1 e jogador 2;
- Lançamento de uma moeda para ver quem é o primeiro a jogar

dfobastos committed:
TicTacToe Project

Primeiros dois passos completos:

- Introduzir o nome do jogador 1 e jogador 2;
- Lançamento de uma moeda para ver quem é o primeiro a jogar

NFSS10 commented:
Noice

NFSS10 commented on TicTacToe.py:
```diff
@@ -0,0 +1,33 @@
+player1 = input("Nome primeiro jogador: ")
+player2 = input("Nome do segundo jogador: ")
+
+lancamento = random.randrange(0,1)
+
+
+if lancamento == 0:
+    player_coin1 = "cara"
+    player_coin2 = "coroa"
+
+else:
+    player_coin2 = "cara"
+    player_coin1 = "coroa"
+
+print(f'{player1} é {player_coin1}')
+print(f'{player2} é {player_coin2}')
+
+lancamento2 = random.randrange(0,1)
```
Qual foi a tua ideia aqui?
Fazes um primeiro lançamento para ver quem fica com cara e com a coroa e depois fazes outro lançamento para ver o que sai e definir quem é o primeiro a jogar?

dfobastos replied:
O primeiro jogador tem a possibilidade de escolher se quer cara ou coroa, se o mesmo escolher cara automaticamente a "coroa" irá ser atribuída ao segundo jogador, e assim vice-versa.

O segundo lançamento serviu para a moeda ser lançada e ver quem vai ser o jogador que irá iniciar.

NFSS10 committed:
Desafio de desenhar o tabuleiro

NFSS10 committed:
Jogador joga à vez

NFSS10 committed:
Ajudas

dfobastos committed:
Jogo_da_adivinhação

Boa tarde pessoal,

Neste exercicio consegui completar 2 dos 3 objectivos pedidos:

Falta apenas validar as 10 tentativas, podem dar uma ajudar?

dfobastos committed:
Jogo de Adivinhação

Boa tarde pessoal,

Completei 2 dos 3 desafios do exercicio, ficando assim a faltar colocar a variavel das "10  tentativas que o jogador tem para acertar".

Se poderem dar umas luzes :)

dfobastos committed:
Conversao de temperatura

Neste exercicio penso que consegui chegar à conclusao, no entanto no "format" do print final algo nao fica bem parametrizado.

NFSS10 commented on DesenharTabuleiro.py:
```diff
@@ -0,0 +1,104 @@
+def alterarEstado1():
+    """
+    board_state tem de representar o seguinte estado:
+    
+       A   B   C
+    1| x |   |   | 
+    2|   | x |   |
+    3|   |   | x |
+
+    """
+    board_state = None
+
+
+
+def alterarEstado2():
+    """
+    board_state tem de representar o seguinte estado:
+    
+       A   B   C
+    1| o |   | x | 
+    2|   | o |   |
+    3|   |   | o |
+
+    """
+    board_state = None
+
+
+def alterarEstado3():
+    """
+    board_state tem de representar o seguinte estado:
+    
+       A   B   C
+    1| x | o | x | 
+    2| o | o | o |
+    3| x | o | x |
+
+    """
+    board_state = None
+
+
```
@dfobastos Então, sempre conseguis-te resolver estas tarefas?
